python-sdk/src/astro/sql/operators/transform.py

```diff
@@ -60,6 +60,7 @@ def execute(self, context: Context):
             target_table=self.output_table,
             parameters=self.parameters,
         )
+        context["ti"].xcom_push(key="output_table_row_count", value=str(self.output_table.row_count))
```

Comment:
does this only need for the transform operator only and not for other operators inherited from `BaseSQLDecoratedOperator`?

Reply:
since if the output table is not passed then BaseSQLDecoratedOperator  assign it to Table()

Reply:
Yes, currently only for this operator open lineage is failing to fetch row_count

Reply:
hmm, I see, look like we do not have lineage for `run_raw_sql`